packages/coinstac-ui/app/render/components/dashboard-home.js

```diff
@@ -2,76 +2,32 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Alert } from 'react-bootstrap';
 import { connect } from 'react-redux';
-import { core, logger, notify } from 'ampersand-app';
-import { flatten, sortBy } from 'lodash';
+import { logger, notify } from 'ampersand-app';
 
 import StatusItem from './status-item.js';
 import { fetch as fetchProjects } from '../state/ducks/projects.js';
 import { fetchComputations } from '../state/ducks/computations.js';
+import { fetch as fetchConsortiaResults } from '../state/ducks/remote-results';
 
 class DashboardHome extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      remoteResults: [],
-    };
-  }
-
   componentWillMount() {
-    const { dispatch } = this.props;
+    const { computations, dispatch, remoteResults } = this.props;
+
+    if (!remoteResults.length) {
+      dispatch(fetchConsortiaResults());
+    }
+
```

Comment:
Using a redux action means that the spinner will appear while the application waits for *remoteResults* to resolve.